rollup/plugin-invariant.js

```diff
@@ -0,0 +1,77 @@
+import { createFilter } from "rollup-pluginutils";
+import recast from "recast";
+
+const b = recast.types.builders;
+
+export default function invariantPlugin(options = {}) {
+  const filter = createFilter(options.include, options.exclude);
+
+  return {
+    transform(code, id) {
+      if (!filter(id)) {
+        return;
```

Comment:
Using a `filter` here is apparently good hygiene / the "right" thing to do when implementing a Rollup transformer plugin: https://rollupjs.org/guide/en#transformers